### ISSUE TITLE
Add caps.

### ISSUE THREAD

**Issue by artelse** [enricoflor/diacritics#1]
Great to see activity on Emacs to ease the use of accented characters. Just this week two packages appeared; this one and [accent.el](https://github.com/elias94/accent/blob/main/accent.el).
Accent.el also defined caps characters to the list and thought you might like to do the same. 
For both would love to see Corfu popup support.
Thx!

**Comment by enricoflor** [comment]
Thank you for your interest! I just pushed the addition of caps, so that they are present by default already. Regarding Corfu, thanks for the suggestion!

I will consider doing this but I think it would require making this a real package. Right now it's more of an exercise to see how one could have Emacs detect long key presses. This is why I wrote it without dependencies and I am not planning to submit it to any package repository. And in fact, it is very hacky and I don't expect it to work very reliably for everyone. Having said that, if it works for you, as it does for me, I am happy!
